casper-application-domain/src/main/kotlin/hs/kr/entrydsm/domain/calculator/aggregates/Calculator.kt

```
@@ -0,0 +1,287 @@
+    /**
+     * 수식에서 사용된 변수들을 추출합니다.
+     *
+     * @param formula 분석할 수식
+     * @return 변수 이름 집합
+     */
+    fun extractVariables(formula: String): Set<String> = try {
+        val tokens = tokenize(formula)
+        tokens.filter { it.type == TokenType.VARIABLE || it.type == TokenType.IDENTIFIER }
+            .map { it.value }
+            .toSet()
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)